From 2e3db17d59143bfaf15423feb7eed68893e04f66 Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.org>
Date: Thu, 24 Oct 2019 14:31:00 +0100
Subject: [PATCH] dts: bcm2838: Disable DWC OTG block by default

Turning off the OTG USB block saves power. Since it requires the use of
the dwc2 overlay to make use of it, we can disable it by default.

Signed-off-by: Phil Elwell <phil@raspberrypi.org>
---
 arch/arm/boot/dts/bcm2838.dtsi | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm/boot/dts/bcm2838.dtsi
+++ b/arch/arm/boot/dts/bcm2838.dtsi
@@ -647,6 +647,7 @@
 
 &usb {
 	interrupts = <GIC_SPI 73 IRQ_TYPE_LEVEL_HIGH>;
+	status = "disabled";
 };
 
 &hdmi {
